src/api/hooks/video.ts

```diff
@@ -32,8 +33,13 @@ export const useVideos = (variables?: GetVideosQueryVariables, opts?: VideosOpts
       where: variables?.where,
     },
   })
+  const videos =
+    variables?.orderBy === VideoOrderByInput.CreatedAtAsc
+      ? data?.videos?.slice()?.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())
+      : data?.videos?.slice()?.sort((a, b) => a.createdAt.getTime() - b.createdAt.getTime())
```

Comment:
I'm not sure if this actually achieves the goal of this issue. It will work fine when you have just one page of videos, but when your videos will span more than one page and you select `Oldest first`, this will still make the pages ordered in the newest first order, but just sort the items on each page. This will need further improvement

Reply:
True, will do it from the read